src/composite-data/hooks/cellar/useCellarData.tsx

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from "@tanstack/react-query"
+import { AaveV2CellarV2 } from "src/abi/types"
+import { fetchCellarData } from "../../actions/cellar/AAVE_V2_STABLE_CELLAR/fetchCellarData"
+import { ContractProps } from "../types"
+
+export const useCellarData = ({
+  cellar,
+}: {
```

Comment:
It's a nitpick, but could we abstract these props to an interface at the top of the file?

```ts
interface UseCellarDataProps {
  cellar: Omit<ContractProps, "signer">
}

export const useCellarData = ({ cellar } : UseCellarDataProps) => {...}
```

same goes for the rest of the hooks in this dir 😁